### PR TITLE
[Needs Review] B06902100

### PR DESCRIPTION
**Pull request by CalvinL15**
A Pull Request (PR) should explain why we need this PR and how you implement it, so others could understand it quickly.
Since this is just a homework, you do not need to explain why and how. 

**Do NOT change the status to NeedsReview if you do not want to be graded yet.**

Please fill in the following questions as feedback. It is totally optional and would not be graded.

## What is the most challenging part in this homework?
environment settings and task 8
## Is this homework helpful for you?
yes in learning Python 
## Is there any suggestion?
no
<br />

### CODE REVIEW

File: src/students/b06902100.py [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+        * Use len() to get the length of the list.
+        * Different data types could not be added. Use str() to convert data
+        to string.
+        * Run `python src/autograder.py -task 2 -student_id <your student ID>`
+        to see if you pass this task.
+        * The correct output would be (5, '1 taiwan')
+    '''
+    # TODO: change length and sentence to fit the requirement
+    length = len(input_list)
+    sentence = str(input_list[0])+input_dictionary["a"]
```
Comment (amjltc295) [inline src/students/b06902100.py]: Hard-coding the keys is not a good solution.

Comment (CalvinL15) [reply src/students/b06902100.py]: I have (hopefully) fixed it

**Comment by amjltc295** [comment]
@CalvinL15 Your commit message is not linked to your profile. See https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/ and https://www.git-tower.com/learn/git/faq/change-author-name-email to fix it.

**Comment by CalvinL15** [comment]
Should I change the author of past commits?

**Comment by amjltc295** [comment]
@CalvinL15 That is up to you.

**Comment by CalvinL15** [comment]
I think it's okay to merge it now

**Pull request merged**